src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -468,8 +468,9 @@ value_set_dereferencet::valuet value_set_dereferencet::build_reference_to(
     }
     else
     {
+      // try to build a member/index expression - do not use byte_extract
       exprt subexpr = get_subexpression_at_offset(
-        root_object_subexpression, o.offset(), dereference_type, ns);
+        root_object_subexpression, o.offset(), dereference_type, "", ns);
```

Comment:
Having an empty string literal, interpreted as an irep ID, to block particular functionality (descending within objects if I understand correctly) does not seem like the most intuitive of designs.  I can't immediately suggest better though.